src/lib.rs

```diff
@@ -47,6 +47,9 @@ use std::iter::{Chain, Rev};
 #[cfg(not(has_extern_crate_alloc))]
 use std::slice::{Iter as SliceIter, IterMut as SliceIterMut};
 
+#[cfg(any(feature = "serde_support", feature = "serde_support_test"))]
```

Comment:
Maybe just add a dependency from the `serde_support_test` feature to the `serde_support` feature?

Reply:
Done.